R/repr.R

```diff
@@ -0,0 +1,54 @@
+### File R/repr.R
+### Part of the R package googleVis
+
+### It is made available under the terms of the GNU General Public
+### License, version 2, or at your option, any later version,
+### incorporated herein by reference.
+###
+### This program is distributed in the hope that it will be
+### useful, but WITHOUT ANY WARRANTY; without even the implied
+### warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+### PURPOSE.  See the GNU General Public License for more
+### details.
+###
+### You should have received a copy of the GNU General Public
+### License along with this program; if not, write to the Free
+### Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
+### MA 02110-1301, USA
+
+#' repr_html.gvis
+#'
+#' Output a gvis chart as an embedded html chart in the Jupyter ecosystem
+#'
+#' @description 
+#' This function lets you use googleVis charts as html output in e.g. a Jupyter Notebook. 
+#'
+#' @param obj A gvis object.
+#' @param ... unused
+#'
+#' @return Returns the html representation of the gvis object as string
+#'
+#' @author Jan Schulz, \email{jasc@gmx.net}
+#'
+#' @name repr_*.gvis
+#'
+#' @keywords jupyter 
+#'
+#' @export
+repr_html.gvis <- function(obj, ...){
+  htmlfile <- tempfile(fileext = '.html')
```

Comment:
This could be easier if the "to string" part of the print.gvis method would be extracted and could then be reused here (and in a `knit_print.gvis` method). I ddin't want to start a refactoring in this PR, but if you want that, I can also add it to this PR.